File: translation_output/codellama/codenet/java/python/p03023.py
<s> [INST] <<SYS>>
You are an expert python programmer and assistant
<</SYS>>

java code:

import java.util.Scanner;

public class p03023 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();

        int answer = 180 * (num - 2);

        System.out.println(answer);
    }
}


Translate the above java code to python.

python code:

[/INST]
```
import math

num = int(input("Enter a number: "))

answer = 180 * (num - 2)

print(answer)
```
In this code, we first import the `math` module, which is not used in the Java code. We then prompt the user to enter a number using the `input()` function, which is similar to the `Scanner` class in Java.

We then calculate the answer using the formula `180 * (num - 2)`, which is the same as the Java code. Finally, we print the answer using the `print()` function.

Note that the `input()` function in Python returns a string, so we need to convert it to an integer using the `int()` function before using it in the calculation.</s>

File: translation_output/codellama/codenet/python/java/p03023.java
<s> [INST] <<SYS>>
You are an expert java programmer and assistant
<</SYS>>

python code:

n = int(input())
s = 180 * (n - 2)
print(s)

Translate the above python code to java.

java code:

[/INST]

Here is the Java code that translates the Python code:

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int s = 180 * (n - 2);
        System.out.println(s);
    }
}

In this code, we first import the `Scanner` class from the `java.util` package, which allows us to read input from the user. We then create a new `Scanner` object and use it to read an integer from the user. We store this integer in a variable called `n`.

Next, we calculate the value of `s` using the formula `180 * (n - 2)`. We then print the value of `s` to the console using the `System.out.println()` method.

Note that the `Scanner` class is used to read input from the user, and the `System.out.println()` method is used to print output to the console.</s>
